Sources/ArcGISToolkit/Compass.swift

```diff
@@ -54,7 +54,7 @@ public class Compass: UIImageView {
         super.init(frame: .zero)
         
         // Set our image to the CompassIcon in the Assets
-        let bundle = Bundle(for: type(of: self))
+        let bundle = Bundle.module
         image = UIImage(named: "CompassIcon", in: bundle, compatibleWith: nil)
```

Comment:
Since the desired bundle is now just a static stored property on `Bundle` itself, it would be simpler just to refer to that:

```suggestion
        image = UIImage(named: "CompassIcon", in: .module, compatibleWith: nil)
```